channels/rdpdr/server/rdpdr_main.c

```diff
@@ -48,7 +47,7 @@ static int rdpdr_server_send_announce_request(RdpdrServerContext* context)
 	Stream_Write_UINT16(s, context->priv->VersionMinor); /* VersionMinor (2 bytes) */
 	Stream_Write_UINT32(s, context->priv->ClientId); /* ClientId (4 bytes) */
 	Stream_SealLength(s);
-	status = WTSVirtualChannelWrite(context->priv->ChannelHandle, (PCHAR) Stream_Buffer(s), Stream_Length(s), &written);
+	WTSVirtualChannelWrite(context->priv->ChannelHandle, (PCHAR) Stream_Buffer(s), Stream_Length(s), &written);
```

Comment:
this is the kind of change I really have a problem with... yes, the status code was obtained but it was unchecked. the fix shouldn't be to remove the status variable, it should be to check the status variable and return -1 on error. I really, really do not want to play cat and mouse in commits where a future commit will just do the exact opposite of such commits + simply check the status variable correctly for error.